docs/logs/discovery/intro.md

```diff
@@ -0,0 +1,22 @@
+title: Logs Discovery
+description: Logs Discovery Introduction
+
+Logs discovery aims to automate the classification and shipping of your infrastructure-crucial log files and container logs. With a simple control plane at your disposal it's easy to explore and manage discovered log sources across bare-metal/virtual machines and containers.
+
+## Features
+
+### Contextual log discoveries
+
+Each log file is linked to the service that's writing to it. Log file metadata reveals some fundamental attributes of a file, like the disk device where it is located, size, inode number and so on.
+
+### Automatic log shipping
+
+Setting up log shipping is straightforward and it boils down to selecting your logs destination.
+
+### Log parsing
```

Comment:
> Though limited at this time, I think this is where they ought to be mentioned.

Should we mention all integrations that we provide parsers for?

Reply:
I don't think we should list them because it would create a maintenance nightmare.  I think we we can adjust the wording and include a link to https://sematext.com/docs/integration/#logging .  That target page/section is not ideal, but we can adjust it to do a better job of featuring logpacks.